### PR TITLE
Improve docs for the getting started guide for oauth2_proxy

### PR DESCRIPTION
**Pull request by philips**
I want to enable people to use Dex with Bitly's OAUTH2 proxy. These are some
docs and fixture improvements to make that work really quickly.

See the PR here: https://github.com/bitly/oauth2_proxy/pull/163

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,49 +1,60 @@
-Running Examples
-===
-
-The quickest way to start experimenting with dex is to run a single dex-worker
-locally, with an in-process database, and then interacting with it using the
-example programs in this directory.
+# Running Examples
 
+The quickest way to start experimenting with dex is to run a single dex-worker locally, with an in-process database, and then interacting with it using the example programs in this directory.
```
Comment (joshix) [inline examples/README.md]: "...and then interact with it..."

Comment (philips) [inline examples/README.md]: fixed

**Comment by philips** [comment]
Can I merge now? :)

**Comment by joshix** [comment]
@philips No you can't, because I already did!

**Pull request merged**